### PR TITLE
[releasenotes] CPO 20240124

### PR DESCRIPTION
**Pull request by BramVerniest**

### CODE REVIEW

**Comment by jeroencosyn** [comment]
@BramVerniest, I can approve, but I think Bruno, Lina, or Sean should approve before merging it. I'm less aware of the changes.

**Review by BrunoVermeersch — APPROVED** [review]

**Pull request merged**
